charts/steward/values.yaml

```diff
@@ -0,0 +1,76 @@
+# Default values for steward.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+runController:
+  image:
+    repository: stewardci/stewardci-run-controller
+    tag: 191217_cc244c0
+    pullPolicy: IfNotPresent
+  resources:
+    limits:
+      cpu: 100m
```

Comment:
In general you want limits.memory == request.memory.
Setting limits.cpu does not really make sens in most situations.

Reply:
> In general you want limits.memory == request.memory.

That should be the case here. Specifying only a limit sets requests = limit (see [docs](https://kubernetes.io/docs/tasks/administer-cluster/manage-resources/memory-default-namespace/#what-if-you-specify-a-container-s-limit-but-not-its-request)).

> Setting limits.cpu does not really make sens in most situations.

It prevents the container from consuming all available CPU capacity of the node, e.g. in case of a malfunction (e.g. endless loops in multiple threads). As we know that our controller doesn't need more that the given amount of CPU, we can set the limit. But for acceptable performance a lower CPU capacity is sufficient, so we set requests to a lower value.

Reply:
1/10 of a cpu for an application with multiple threads as a hard limit?
I would be surprised if this does not artificially throttle the performance of the application.

Reply:
I've set 1 CPU as a limit. Let's try with this and observe the metrics.